skyvern/forge/agent.py

```diff
@@ -791,6 +791,11 @@ async def _build_and_record_step_prompt(
             artifact_type=ArtifactType.VISIBLE_ELEMENTS_TREE_TRIMMED,
             data=json.dumps(scraped_page.element_tree_trimmed, indent=2).encode(),
         )
+        await app.ARTIFACT_MANAGER.create_artifact(
+            step=step,
+            artifact_type=ArtifactType.VISIBLE_ELEMENTS_TREE_IN_PROMPT,
+            data=scraped_page.build_element_tree(element_tree_format).encode(),
```

Comment:
this and line 772 is duplicated. we can set a variable `element_tree_in_prompt` before line 772 and reuse that variable